ui-tests/src/test/java/io/syndesis/qe/fragments/common/menu/KebabMenu.java

```diff
@@ -40,7 +40,7 @@ public void close() {
 
     public SelenideElement getItemElement(String item) {
         ElementsCollection found = menuButton.parent().$$(By.cssSelector("a")).filter(Condition.exactText(item));
-        found = found.exclude(Conditions.NO_CHILDREN);
```

Comment:
hi @mkralik3 , could you check please, whether this condition should not be rather HAS_CHILDREN on other places as well? Because NO_CHILDREN option excudes all elements without children, e.g. "Delete" option of kebab menu.

Reply:
@Stefan365 
NO_CHILDREN was used because connection kebab menu has two links nested... so changing this would fix databucket but break connections. Could you perhaps rewrite the NO_CHILDREN condition to allow for specifying a selector, so the kebab menu could ignore links with nested links?